app-local-first-react-router/src/routes/connexion/creation-de-compte.tsx

```diff
@@ -0,0 +1,172 @@
+import { Input } from '@codegouvfr/react-dsfr/Input';
+import { Link, useSearchParams, useNavigate } from 'react-router';
+import { Button } from '@codegouvfr/react-dsfr/Button';
+import { PasswordInput } from '@codegouvfr/react-dsfr/blocks/PasswordInput';
+import { getUserOnboardingRoute } from '@app/utils/user-onboarded.client';
+import { CallOut } from '@codegouvfr/react-dsfr/CallOut';
+import { type User } from '@prisma/client';
+import { type UserConnexionResponse } from '@api/src/types/responses';
+import { useEffect, useState } from 'react';
+import { refreshUser } from '@app/utils-offline/get-most-fresh-user';
+import Chargement from '@app/components/Chargement';
+import { capture } from '@app/services/sentry';
+import useUser from '@app/zustand/user';
+import useZustandStore from '@app/zustand/store';
+import API from '@app/services/api';
+import { clearCache } from '@app/services/indexed-db';
+
+export default function CreationDeCompte() {
+  const [searchParams] = useSearchParams();
+  const [initialLoading, setInitialLoading] = useState(true);
```

Comment:
```suggestion
  const [initialLoading, setInitialLoading] = useState(!!user);
```

je mettrai ça
- si y'a pas de user, y'a pas besoin d'afficher "Chargement" : c'est une création de compte
- si y'a un user, alors tu clear le cache / checke s'il est légitime / etc., donc y'a bien chargement